did_parser/examples/did_parser/main.rs

```diff
@@ -0,0 +1,10 @@
+use did_parser::{Did, DidUrl};
+
+fn main() {
+    // parse a string into DID
+    Did::parse("did:web:w3c-ccg.github.io".into()).expect("should correctly parse did");
```

Comment:
Lets not use `expects` under examples - it distorts the attention toward unhappy path which is not expected to happen in examples. Lets assume the implementation is actually correct (this crate is well unit tested)

Secondly I would suggest instead `println` the parsed DIDs - so if I am newcomer and run examples, I can see some interesting output.